jwt/api_jws.py

```diff
@@ -95,7 +95,7 @@ def encode(self, payload, key, algorithm='HS256', headers=None,
         signing_input = b'.'.join(segments)
         try:
             alg_obj = self._algorithms[algorithm]
-            key = alg_obj.prepare_key(key)
+            key = alg_obj.prepare_key(key, **kwargs)
```

Comment:
Well, there were no `kwargs` at all until this point. It could be considered to have one argument `password` instead. Don't have much time for now.